middleware/checkUserCredentials.js

```diff
@@ -0,0 +1,11 @@
+const checkUserCredentials = (request, response) => {
+  const { username, password } = request.body;
+
+  if (username === "lee" && password === "marsrover") {
+    response.send("You are now signed in");
```

Comment:
What's the actual UX flow for the user when they log in?

When they submit the form, what do they see? How does logging in work on the websites we normally use day to day?

Reply:
Ah good shout, I'll add logic to redirect the user to the home page once they are signed in